### PR TITLE
🏁 Dome EOL: deprecate and disable bottles

### PR DESCRIPTION
**Pull request by chapulina**
* Part of https://github.com/ignition-tooling/release-tools/issues/600

### CODE REVIEW

**Review by j-rivero — APPROVED** [review]

**Comment by scpeters** [comment]
I need to check if we can still build bottles after this has been merged; we may need to revert until all the dome releases are done

File: Formula/ignition-dome.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@ class IgnitionDome < Formula
     sha256 cellar: :any, mojave:   "25f52cf2b4893b2e98c3a9da733c33c434aa6eb9565f34bead4fb4eb725d6854"
   end
 
+  disable! date: "2022-01-31", because: "is past end-of-life date"
```
Comment (scpeters) [inline Formula/ignition-dome.rb:19]: adding a `disable!` tag automatically deprecates immediately

Comment (chapulina) [reply Formula/ignition-dome.rb:19]: * So you're saying that the date isn't respected? I was following what we did for Blueprint: https://github.com/osrf/homebrew-simulation/pull/1242

Comment (chapulina) [reply Formula/ignition-dome.rb:19]: * And now I see that was reverted in https://github.com/osrf/homebrew-simulation/pull/1256 :upside_down_face:

**Comment by chapulina** [comment]
> I need to check if we can still build bottles after this has been merged

* Ah now tracing back what we did for Blueprint, I think it may not work: https://github.com/ignitionrobotics/sdformat/pull/449#issuecomment-752300521

File: Formula/ignition-dome.rb (hunk 1 above)
Comment (chapulina) [reply Formula/ignition-dome.rb:19]: https://github.com/osrf/homebrew-simulation/pull/1737

**Pull request merged**